### PR TITLE
modules.ceph_cfg.init: Add cephfs methods

### PR DESCRIPTION
**Pull request by oms4suse**
Adding the follosing cephfs methods:

```
cephfs_list
cephfs_add
cephfs_del
```

Signed-off-by: Owen Synge osynge@suse.com

### CODE REVIEW

**Comment by jan--f** [comment]
The Notes sections in all three docs is unclear. See https://github.com/oms4suse/sesceph/pull/39/commits/8a8379433dadc95d39108c81ccba3abb276cce27#diff-2521e27525ecb75b3a0998409aa34984R1384 for example
I guess it's a copy/paste error?

**Comment by oms4suse** [comment]
Improved comments.

File: _modules/ceph_cfg/__init__.py [hunk 1]
```diff
@@ -1367,3 +1367,103 @@ def cluster_status(**kwargs):
...
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' ceph_cfg.cephfs_del \\
+                del_cephfs \\
+                'cluster_name'='ceph' \\
+                'cluster_uuid'='cluster_uuid'
+    Notes:
+    All mon nodes should be stopped to allow thsi operation to success.
```
Comment (jan--f) [inline _modules/ceph_cfg/__init__.py]: looks good...just a tiny typo here

**Comment by jan--f** [comment]
looks good! just a tiny typo here https://github.com/oms4suse/sesceph/pull/39/files#diff-2521e27525ecb75b3a0998409aa34984R1453

**Comment by jan--f** [comment]
Reviewed-by: Jan Fajerski jfajerski@suse.com

**Pull request merged**